nss-day-cohort-47/holiday-road-grand-teton-titans
# Bk alert boxes

# Description

Created event listener in main.js that invokes showDetails() in ParkPreview.js when 'details' button is clicked in the itinerary preview box
-showDetails() function has not been completed


## Type of change




# Testing Instructions

Select a national park from the drop down, then click 'details' button
console should stop at showDetails()

# Checklist:
